src/main.py

```diff
@@ -98,18 +98,23 @@
     # such as unsafe-inline or unsafe-eval is highly discouraged, as these
     # may lead to XSS attacks.
     'csp_policy': {
-        # https://developers.google.com/fonts/docs/technical_considerations
-        'font-src':    '\'self\' themes.googleusercontent.com '
-                       '*.gstatic.com',
+        # Fallback.
+        'default-src': '\'self\'',
+        # Disallow Flash, etc.
+        'object-src': '\'none\'',
+        # Google Analytics.
+        'script-src': '\'self\' https://www.google-analytics.com/analytics.js',
         # Maps, YouTube provide <iframe> based embedding at these URIs.
-        'frame-src':   '\'self\' www.google.com www.youtube.com',
-        # Assorted Google-hosted APIs.
-        'script-src':  '\'self\' *.googleanalytics.com *.google-analytics.com',
+        'child-src':  '\'self\' https://www.google.com https://www.youtube.com',
+        # Deprecated. Used for supporting browsers that use CSP 1.0 only.
+        'frame-src':  '\'self\' https://www.google.com https://www.youtube.com',
```

Comment:
Not sure. I guess this should be the developers decision which cutoff percentage is acceptable for his use case. 
I would rather keep the deprecated directive (which will be ignored by newer browsers) longer than breaking functionality in older browsers.